gui/re.chiaki.Chiaki.appdata.xml

```diff
@@ -14,9 +14,12 @@
       Chiaki is a Free and Open Source Client for PlayStation 4 Remote Play. It can be used to play in real time on a PlayStation 4 as long as there is a network connection.
     </p>
     <p>
-      For more information and instructions how to use Chiaki, please refer to the official documentation: https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage
+      For more information and instructions how to use Chiaki, please refer to the official documentation: <url type="help">https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage</url>
```

Comment:
As far as I know, actual urls are **not supported** in the description section. If you want to add the "help" url, feel free to add it below the `<url type="homepage">` url before description section, but please, **do not remove or change** the text in description because, if I am not mistaken, "help" urls are not displayed in GNOME Software at all.

Reply:
XML specification doesn't allow to use URIs without tags. That's why I used it. Validation goes fine with it but if you say it's not supported then sure we can move it to separate tag and just reference it in description.

Reply:
Then remove the whole "<p>For more information and instructions how to use Chiaki, please refer to the official documentation: <url type="help">https://github.com/thestr4ng3r/chiaki/blob/master/README.md#usage</url></p>" part and move the url outside of description section.